### DISCUSSION TITLE
How to input 2 or more different inAnyCollection ids

### DISCUSSION THREAD

**Discussion by nhocleopro** [Shopify/function-examples#410] [Feedback: Discounts APIs]
Hello!

I hope you're having a great day!

I have a question: If I have two volume discount rules:

Rule 1: In collection A
Rule 2: In collection B
Can I input two variables for the two rules to run.graphql?

Reference link: https://tppr.me/Y7sSky

**Comment by nickwesselman** [comment]
In this case, I'd recommend using `inCollections` on [`Product`](https://shopify.dev/docs/api/functions/reference/product-discounts/graphql/common-objects/product) instead to allow you to check the status of each individual collection.

If this doesn't work for some reason, please reopen and further describe your use case.

**Comment by divyansh122** [comment]
I am having difficulty that even if my product exist in the collection still inAnyCollection is returning false , and yes the product and collection both are live on the online store . Anyone know the solution for this ?
